pages/applications/view/[id]/[person]/index.tsx

```diff
@@ -41,40 +51,91 @@ export default function ApplicationPersonPage({
     ? data.mainApplicant
     : data.otherMembers?.find((x) => x.person?.id === person);
 
+  type State =
+    | 'identity'
+    | 'livingsituation'
+    | 'money'
+    | 'health'
+    | 'checklist';
+  const [state, setState] = useState<State>('identity');
+
+  const personalDetails = personalDetailsCheckboxList(applicant);
+  const immigrationStatus = immigrationStatusCheckboxList(applicant);
+  const livingSituation = livingSituationCheckboxList(applicant);
+  const addressHistory = addressHistoryCheckboxList(applicant);
+  const currentAccomodation = currentAccomodationCheckboxList(applicant);
+  const situation = situationCheckboxList(applicant);
+  const employment = employmentCheckboxList(applicant);
+  const incomeAndSavings = incomeAndSavingsCheckboxList(applicant);
+  const medicalDetails = medicalDetailsPageData(applicant);
+
   return (
     <UserContext.Provider value={{ user }}>
       <Layout>
-        <HeadingOne
```

Comment:
This should be kept? we need a heading for the page.

Reply:
This has been re-added